libasynql/src/poggit/libasynql/base/DataConnectorImpl.php

```diff
@@ -151,8 +152,8 @@ public function asyncChange(string $queryName, array $args = []) : Generator{
 		$onSuccess = yield Await::RESOLVE;
 		$onError = yield Await::REJECT;
 		$this->executeChange($queryName, $args, $onSuccess, $onError);
-		$affectedRows = yield Await::ONCE;
-		return $affectedRows;
+		//Return $affectedRows
+		return yield Await::ONCE;
 	}
```

Comment:
since it's two lines anyway, why not make it self-explanatory instead of using a separate comment?